src/main/java/gregtech/common/covers/CoverFluidFilter.java

```diff
@@ -180,7 +181,7 @@ public ModularPanel buildUI(SidedPosGuiData guiData, PanelSyncManager guiSyncMan
                         .child(new Rectangle().setColor(UI_TEXT_COLOR).asWidget()
                                 .height(1).widthRel(0.95f).margin(0, 4))
                         .child(getFilter().createWidgets(guiSyncManager)))
-                .child(SlotGroupWidget.playerInventory().bottom(7).left(7));
+                .child(SlotGroupWidget.playerInventory(false).bottom(7).left(7));
```

Comment:
This should have the player inventory displayed right? So you can set the filter from things inside the inventory?